packages/modelingcovid-covidmodel/connector.wl

```diff
@@ -228,7 +292,7 @@ Print["Adding a custom scenario and distancing function to the precomputed data"
 (* For simplicity, remove all other scenarios,
 except scenario1 which is needed for fitting.
 scenarios=Append[scenarios, customScenario]; *)
-scenarios={scenario1, customScenario};
+scenarios={scenario1, scenario5, customScenario};
```

Comment:
Maybe the comment needs to be updated here? Not sure why `scenario5` is included.